_layouts/integration.html

```diff
@@ -20,19 +20,32 @@ <h2>
   {% endif %}
 </p>
 
+<p>
+    {% if page.quality_estimation_integrations.size == 1 %} 
+      The <strong>{{ page.title }}</strong> {{ page.type_description }} has <a href="#qe-integrations">{{ page.quality_estimation_integrations | size }} quality estimation API integration</a>.
+    {% if page.quality_estimation_integrations > 1 %}
```

Comment:
I guess there should be an `s` here at the end.

I'd probably combine them:

```
The <strong>{{ page.title }}</strong> {{ page.type_description }} has <a href="#qe-integrations">{{ page.quality_estimation_integrations | size }} quality estimation API {% if page.quality_estimation_integrations == 1 %}integration{% else %}integrations{% endunless %}</a>.
```

I think it's fine to let it explicitly say that it has 0 if it has 0, because that will soon be pretty rare for a major TMS.  I can add more data soon.

Reply:
or just `s` like below.